daisy_workflows/build-publish/enterprise_linux/rocky_linux_8_optimized_gcp_with_nvidia_latest.publish.json

```diff
@@ -33,9 +33,9 @@
   {{$time := trimPrefix .publish_version "v"}}
   "Images": [
     {
-      "Prefix": "rocky-linux-8-optimized-gcp-with-nvidia-550",
-      "Family": "rocky-linux-8-optimized-gcp-with-nvidia-550",
-      "Description": "Rocky Linux 8 optimized for GCP with Nvidia 550 built on {{$time}}",
+      "Prefix": "rocky-linux-8-optimized-gcp-nvidia-latest",
+      "Family": "rocky-linux-8-optimized-gcp-nvidia-latest",
+      "Description": "Rocky Linux 8 optimized for GCP with latest LTS Nvidia driver built on {{$time}}",
```

Comment:
Not necessarily latest LTS actually, just latest

Reply:
Fixed